### PR TITLE
Add Helm chart

### PR DESCRIPTION
**Pull request by tuommaki**
Base has been copied from endpoint-operator and then modified for
cluster-operator. This has been tested with small changes (filled docker
config to values.yaml and used latest version for cluster-operator
container image version) in local minikube.

### CODE REVIEW

**Comment by xh3b4sd** [comment]
CI is failing.

File: helm/cluster-operator-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cluster-operator
+  namespace: giantswarm
+  labels:
+    app: cluster-operator
+spec:
+  replicas: 2
```
Comment (MarcelMue) [inline helm/cluster-operator-chart/templates/deployment.yaml]: Why are there two replicas of this operator?

Comment (xh3b4sd) [reply helm/cluster-operator-chart/templates/deployment.yaml]: Good catch. I missed this. The operator should only be there ones.

Comment (tuommaki) [reply helm/cluster-operator-chart/templates/deployment.yaml]: Argh. Forgot that. I already changed it to one, but from the version where I had fixed version for container image ( for local testing). After that I copied the original w/ [[ .SHA ]] which then overwrote the replica count. Just a sec, I'll fix. :)

**Pull request merged**